MediaBrowser.Model/Drawing/ImageDimensions.cs

```diff
@@ -41,5 +68,10 @@ public override string ToString()
                 Width,
                 Height);
         }
+
+        public override int GetHashCode()
+        {
+            return (Height.GetHashCode() * 17) + Width.GetHashCode();
```

Comment:
There's a built-in for this
```suggestion
            return HashCode.Combine(Height, Width);
```